I'm getting this in resultsdb's logs:
```
[2025-05-13 09:40:02] [9] [ERROR] resultsdb: Internal error: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
[2025-05-13 09:40:06] [9] [ERROR] resultsdb: Exception on /api/v2.0/results/latest [GET]
Traceback (most recent call last):
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/base.py", line 1964, in _exec_single_context
    self.dialect.do_execute(
    ~~~~~~~~~~~~~~~~~~~~~~~^
        cursor, str_statement, effective_parameters, context
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/default.py", line 945, in do_execute
    cursor.execute(statement, parameters)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "result" does not exist
LINE 2: FROM result JOIN result_data AS result_data_1 ON result.id =...
             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/venv/lib64/python3.13/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "/venv/lib64/python3.13/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/venv/lib64/python3.13/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/venv/lib64/python3.13/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/venv/lib64/python3.13/site-packages/flask_pydantic/core.py", line 285, in wrapper
    res = func(*args, **kwargs)
  File "/venv/lib64/python3.13/site-packages/resultsdb/controllers/api_v2.py", line 480, in get_results_latest
    results = q.all()
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/orm/query.py", line 2704, in all
    return self._iter().all()  # type: ignore
           ~~~~~~~~~~^^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/orm/query.py", line 2858, in _iter
    result: Union[ScalarResult[_T], Result[_T]] = self.session.execute(
                                                  ~~~~~~~~~~~~~~~~~~~~^
        statement,
        ^^^^^^^^^^
        params,
        ^^^^^^^
        execution_options={"_sa_orm_load_options": self.load_options},
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/orm/session.py", line 2365, in execute
    return self._execute_internal(
           ~~~~~~~~~~~~~~~~~~~~~~^
        statement,
        ^^^^^^^^^^
    ...<4 lines>...
        _add_event=_add_event,
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/orm/session.py", line 2251, in _execute_internal
    result: Result[Any] = compile_state_cls.orm_execute_statement(
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        self,
        ^^^^^
    ...<4 lines>...
        conn,
        ^^^^^
    )
    ^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/orm/context.py", line 306, in orm_execute_statement
    result = conn.execute(
        statement, params or {}, execution_options=execution_options
    )
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/base.py", line 1416, in execute
    return meth(
        self,
        distilled_parameters,
        execution_options or NO_OPTIONS,
    )
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/sql/elements.py", line 523, in _execute_on_connection
    return connection._execute_clauseelement(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        self, distilled_params, execution_options
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/base.py", line 1638, in _execute_clauseelement
    ret = self._execute_context(
        dialect,
    ...<8 lines>...
        cache_hit=cache_hit,
    )
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/base.py", line 1843, in _execute_context
    return self._exec_single_context(
           ~~~~~~~~~~~~~~~~~~~~~~~~~^
        dialect, context, statement, parameters
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/base.py", line 1983, in _exec_single_context
    self._handle_dbapi_exception(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        e, str_statement, effective_parameters, cursor, context
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
ng-booster-2.27.52-1.fc41&type=koji_build%2Cbrew-build&_distinct_on=scenario%2Csystem_architecture%2Csystem_variant HTTP/1.1" 500 36
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/base.py", line 2352, in _handle_dbapi_exception
    raise sqlalchemy_exception.with_traceback(exc_info[2]) from e
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/base.py", line 1964, in _exec_single_context
    self.dialect.do_execute(
    ~~~~~~~~~~~~~~~~~~~~~~~^
        cursor, str_statement, effective_parameters, context
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/venv/lib64/python3.13/site-packages/sqlalchemy/engine/default.py", line 945, in do_execute
    cursor.execute(statement, parameters)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "result" does not exist
LINE 2: FROM result JOIN result_data AS result_data_1 ON result.id =...
             ^

[SQL: SELECT DISTINCT ON (result.testcase_name, result_data_0_scenario.value, result_data_1_system_architecture.value, result_data_2_system_variant.value) result.id AS result_id, result.testcase_name AS result_testcase_name, result.submit_time AS result_submit_time, result.outcome AS result_outcome, result.note AS result_note, result.ref_url AS result_ref_url 
FROM result JOIN result_data AS result_data_1 ON result.id = result_data_1.result_id JOIN result_data AS result_data_2 ON result.id = result_data_2.result_id LEFT OUTER JOIN (SELECT result_data.id AS id, result_data.result_id AS result_id, result_data.key AS key, result_data.value AS value 
FROM result_data 
WHERE result_data.key = %(key_1)s) AS result_data_0_scenario ON result.id = result_data_0_scenario.result_id LEFT OUTER JOIN (SELECT result_data.id AS id, result_data.result_id AS result_id, result_data.key AS key, result_data.value AS value 
FROM result_data 
WHERE result_data.key = %(key_2)s) AS result_data_1_system_architecture ON result.id = result_data_1_system_architecture.result_id LEFT OUTER JOIN (SELECT result_data.id AS id, result_data.result_id AS result_id, result_data.key AS key, result_data.value AS value 
FROM result_data 
WHERE result_data.key = %(key_3)s) AS result_data_2_system_variant ON result.id = result_data_2_system_variant.result_id 
WHERE result_data_1.key = %(key_4)s AND result_data_1.value IN (%(value_1_1)s) AND result_data_2.key = %(key_5)s AND result_data_2.value IN (%(value_2_1)s, %(value_2_2)s) ORDER BY result.testcase_name, result_data_0_scenario.value, result_data_1_system_architecture.value, result_data_2_system_variant.value, result.submit_time DESC]
[parameters: {'key_1': 'scenario', 'key_2': 'system_architecture', 'key_3': 'system_variant', 'key_4': 'item', 'key_5': 'type', 'value_1_1': 'ibus-typing-booster-2.27.52-1.fc41', 'value_2_1': 'koji_build', 'value_2_2': 'brew-build'}]
(Background on this error at: https://sqlalche.me/e/20/f405)
```

Looks like the ResultsDB is not up-to-date? I'll see if I can run a migration script.

--- a/devel/ci/integration/greenwave/settings.py
+++ b/devel/ci/integration/greenwave/settings.py
@@ -8,7 +8,7 @@ KOJI_BASE_URL = 'https://koji.fedoraproject.org/kojihub'
 
 SECRET_KEY = 'this-is-only-for-integration-testing'
 WAIVERDB_API_URL = 'http://waiverdb:8080/api/v1.0'
-RESULTSDB_API_URL = 'http://resultsdb/resultsdb/api/v2.0'
+RESULTSDB_API_URL = 'http://resultsdb:5001/api/v2.0'
 CORS_URL = 'https://bodhi'
 CACHE = {
     "backend": "dogpile.cache.memory_pickle",

--- a/devel/ci/integration/ipsilon/Dockerfile
+++ b/devel/ci/integration/ipsilon/Dockerfile
@@ -1,4 +1,4 @@
-FROM quay.io/fedora/fedora:38
+FROM quay.io/fedora/fedora:latest
 # should be specified as hostname:port to listen on a non-standard port
 ARG hostname=id.dev.fedoraproject.org
 # can be a comma-separated list
@@ -22,7 +22,7 @@ RUN ipsilon-server-install --root-instance --secure no --testauth yes --testauth
 #     --subject-alt-name ipsilon.ci \
 #     --subject-alt-name ipsilon
 RUN \
-    dnf -y install openssl && \
+    dnf -y install gawk openssl && \
     cd /root && \
     curl -L -o easyrsa.tgz https://github.com/OpenVPN/easy-rsa/releases/download/v3.2.2/EasyRSA-3.2.2.tgz && \
     tar -xf /root/easyrsa.tgz && \

--- a/devel/ci/integration/rabbitmq/Dockerfile
+++ b/devel/ci/integration/rabbitmq/Dockerfile
@@ -1,4 +1,4 @@
-FROM quay.io/fedora/fedora:38
+FROM quay.io/fedora/fedora:41
 LABEL \
   name="rabbitmq" \
   vendor="Fedora Infrastructure" \

--- a/devel/ci/integration/resultsdb/Dockerfile
+++ b/devel/ci/integration/resultsdb/Dockerfile
@@ -1,22 +1,11 @@
-FROM quay.io/fedora/fedora:37
+FROM quay.io/factory2/resultsdb:prod-fedora
 LABEL \
   name="resultsdb" \
   vendor="Fedora Infrastructure" \
-  maintainer="Aurelien Bompard <abompard@fedoraproject.org>" \
+  maintainer="Mattia Verga <mattia@fedoraproject.org>" \
   license="MIT"
-
-# Install deps
-RUN dnf install -y \
-    httpd \
-    python3-mod_wsgi \
-    resultsdb \
-    python3-psycopg2 \
-    python3-fedora-messaging
 
 # Configuration
 COPY devel/ci/integration/resultsdb/settings.py /etc/resultsdb/settings.py
-COPY devel/ci/integration/resultsdb/httpd.conf /etc/httpd/conf.d/resultsdb.conf
 COPY devel/ci/integration/resultsdb/fedora-messaging.toml /etc/fedora-messaging/config.toml
 
-EXPOSE 80
-CMD ["/usr/sbin/httpd", "-DFOREGROUND"]

--- a/devel/ci/integration/resultsdb/settings.py
+++ b/devel/ci/integration/resultsdb/settings.py
@@ -1,5 +1,7 @@
 SECRET_KEY = 'this-is-only-for-integration-testing'
 SQLALCHEMY_DATABASE_URI = 'postgresql+psycopg2://resultsdb@db/resultsdb'
+HOST = '0.0.0.0'
+PORT = 5001
 FILE_LOGGING = False
 LOGFILE = '/var/log/resultsdb/resultsdb.log'
 SYSLOG_LOGGING = False

--- a/devel/ci/integration/tests/fixtures/resultsdb.py
+++ b/devel/ci/integration/tests/fixtures/resultsdb.py
@@ -50,8 +50,8 @@ def resultsdb_container(docker_backend, docker_network, db_container, rabbitmq_c
     container = image.run_via_binary(additional_opts=run_opts)
     container.start()
     # Add sample data in the database
-    container.execute(["resultsdb", "init_db"])
+    container.execute(["/venv/bin/resultsdb", "init_db"])
     # we need to wait for the webserver to start serving
-    container.wait_for_port(80, timeout=30)
+    container.wait_for_port(5001, timeout=30)
     yield container
     stop_and_delete(container)

--- a/devel/ci/integration/waiverdb/settings.py
+++ b/devel/ci/integration/waiverdb/settings.py
@@ -1,6 +1,6 @@
 DATABASE_URI = 'postgresql+psycopg2://waiverdb@db:5432/waiverdb'
 SECRET_KEY = 'this-is-only-for-integration-testing'
-RESULTSDB_API_URL = 'https://resultsdb:8080/api/v2.0'
+RESULTSDB_API_URL = 'http://resultsdb:5001/api/v2.0'
 CORS_URL = 'https://bodhi'
 
 # TODO: integration testing of published fedmsgs?

